tests/mgbench/benchmark.py

```diff
@@ -136,7 +94,7 @@ def parse_args():
     benchmark_parser.add_argument(
         "--single-threaded-runtime-sec",
         type=int,
-        default=10,
+        default=1,
```

Comment:
You only changed the runtime in seconds but didn't change the `--query-count-lower-bound` it was 30 before?

Reply:
Yes and I don't know if you checked grafana for this PR but after running each query for 1s, almost all results changed for in memory transactional. So probably 1s is not enough. But I am trying to find some good point for which mgbench won't be run for to long while not losing precision. Any advices?

Reply:
I have seen, but I have noticed that @gitbuda and you are having a discussion regarding that. 

Import is 10-12 minutes per workload = 30 minutes if you run all workloads.  We can export snapshot datasets and inject them via custom import from S3. 

Expansion 4 took a lot of time (3-4 min I guess) and provides low value since this is a social network. You are returning almost the whole graph, due to the density of the graph. But on the other hand, it is the only query that stresses memgraph memory usage and performance. 

Those are two options that come to my mind.

Reply:
One more option: maybe we could define a smaller group of queries that we run on the diff, for example, `pokec*/medium/diff`, and then have a chron job running the `pokec*/medium/daily` that would involve a much bigger set of queries since they should only grow in size.